docs/100-upgrade/migration-guides/3.3-3.4.mdx

```diff
@@ -438,3 +438,83 @@ index 19f85b84e..954bbb896 100644
 ```
 
 </details>
+
+## From `graphql.unstable_module` to `graphql.module`
+
+In this new Front-Commerce version we've created a brand new GraphQL module API,
+here's a guide to help you migrate to the new API:
+
+### Runtime
+
+Runtime is now the core of GraphQL modules, you can check the
+createGraphQLRuntime documentation to learn more about this. The migration is
```

Comment:
**suggestion:** Add a link to the related documentation.

Reply:
Done